press/scripts/migrate.py

```diff
@@ -321,42 +325,84 @@ def is_valid_subdomain(subdomain):
 
 
 @add_line_after
-def upload_backup(local_site):
-	# take backup
-	files_uploaded = {}
-	print("Taking backup for site {}".format(local_site))
+def take_backup(local_site):
+	print(f"Taking backup for site {local_site}")
 	odb = frappe.utils.backups.new_backup(ignore_files=False, force=True)
+	return [
+		(
+			"config",
+			getattr(odb, "site_config_backup_path", None)
```

Comment:
use a separate variable for this, if your line is folding it's too long to be an expression